Alignment/TrackerAlignment/test/cosmicRateAnalyzer_cfg.py

```diff
@@ -0,0 +1,36 @@
+# Implementation : Description of running this tool can be find  at :
+# https://twiki.cern.ch/twiki/bin/view/CMS/TkAlCosmicsRateMonitoring
+
+import FWCore.ParameterSet.Config as cms
+
+import os
+
+process = cms.Process("cosmicRateAnalyzer")
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+
+process.load('Configuration.StandardSequences.MagneticField_cff') # B-field map
+process.load('Configuration.Geometry.GeometryRecoDB_cff') # Ideal geometry and interface
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff") # Global tag
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag.globaltag = GlobalTag("80X_dataRun2_Prompt_v2")
```

Comment:
@skeshri minor comment, since this is an example, you might want to remove the reference to a specific GT and make use of a autoCond key, like `auto:run2_data`